testfiles/shelltests/s3mMultipleLanguagesTests.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/sh
+# file: s3mMultipleFileTypesTests.sh
+
+. ./functions/s3mMultipleLanguagesFunctions.sh
+
+# Test the tool with C++ files.
+testCPP() {
+
+	multipleLanguages "$START_PATH/otherlanguages" "cpp"
+}
+
+# Test the tool with Typescript files.
+testTS() {
+
+	multipleLanguages "$START_PATH/otherlanguages" "ts"
+}
+
+suite_addTest testCPP
+suite_addTest testTS
```

Comment:
But this only works if all of the tests are in the same file. If they are distributed in different files (our organization), this won't work. Besides, they're technically being called from within the suite function, since there's an eval there.